Changelog.md

```diff
@@ -2,6 +2,7 @@
 
 Features:
  * Parser: Better error message for unexpected trailing comma in parameter lists.
+ * Type Checker: Overload resolution should work with builtins of ``address``.
```

Comment:
I think this is rather about overwriting instead of overloading. What about `Fix virtual function lookup when overwriting builtin members of ``address``.`?